assets/queries/terraform/azure/network_interfaces_dont_use_public_IPs/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	network := input.document[i].resource.azurerm_network_interface[name].ip_configuration
+
+	common_lib.valid_key(network, "public_ip_address_id")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_network_interface[%s].ip_configuration.public_ip_address_id", [name]),
```

Comment:
```suggestion
		"searchKey": sprintf("azurerm_network_interface.%s.ip_configuration.public_ip_address_id", [name]),
```

Reply:
I think you should keep the [] in this case

Reply:
Yes, It was a misleading by my side, this suggestion should be ignored